pingaccess/resource_pingaccess_load_balancing_strategy.go

```diff
@@ -0,0 +1,125 @@
+package pingaccess
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/iwarapter/pingaccess-sdk-go/pingaccess"
+)
+
+func resourcePingAccessLoadBalancingStrategy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourcePingAccessLoadBalancingStrategyCreate,
+		Read:   resourcePingAccessLoadBalancingStrategyRead,
+		Update: resourcePingAccessLoadBalancingStrategyUpdate,
+		Delete: resourcePingAccessLoadBalancingStrategyDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: resourcePingAccessLoadBalancingStrategySchema(),
+	}
+}
+
+func resourcePingAccessLoadBalancingStrategySchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		className: &schema.Schema{
+			Type:         schema.TypeString,
+			Required:     true,
+			ValidateFunc: validateClassNameValue,
```

Comment:
This won't work, the Load Balancing Strategies is one of the parts of the SDK and as such people can create their own classes to implement whatever logic they require, as we can never know what these names would be it's best to remove the validator.